content/src/fundings/dep-clean-construction.md

```diff
@@ -25,15 +25,17 @@ sector:
   - utilities
 openDate: ""
 dueDate: ""
-descriptionMd: "New Jersey Clean Construction Program installs pollution control devices or replaces qualifying non-road construction equipment that reduces pollution. Businesses can apply to two grants: Off-Road Diesel Retrofit Grant and Off-Road Diesel Replacement Grant."
+descriptionMd: >-
+  New Jersey Clean Construction Program installs pollution control devices or replaces qualifying non-road
+  construction equipment that reduces pollution. Businesses can apply to two grants: Off-Road Diesel Retrofit
+  Grant and Off-Road Diesel Replacement Grant.
+summaryDescriptionMd: >-
+  New Jersey Clean Construction Program installs pollution control devices or replaces qualifying non-road
+  construction equipment that reduces pollution. Businesses can apply to two grants: Off-Road Diesel Retrofit
+  Grant and Off-Road Diesel Replacement Grant.
```

Comment:
[dust] interesting that these are the same in multiples places, maybe a refactor candidate to consolidate somehow?

Reply:
This may have something to do with the Webflow syncing and the fact the body doesn't get rendered on Webflow and the summaryDescriptionMd doesn't get rendered in Navigator?